java/src/main/java/com/training/db/hw2/Factorial.java

```diff
@@ -0,0 +1,38 @@
+package com.training.db.hw2;
+
+public class Factorial {
+
+    public static void calculateFactorial(int n2_fac) {
+
+        int n3 = UiUtil.input("Input third number: ");
```

Comment:
as n2 it should be in main func and passed to func as param.

Reply:
done